rules/do-you-know-how-to-use-single-codebase-for-multiple-domains-with-tinacm-and-nextjs/rule.md

```diff
@@ -0,0 +1,163 @@
+---
+type: rule
+tips: ""
+title: Do you know how to use Single Codebase for Multiple Domains with TinaCMS
+  and Next.js
+seoDescription: Optimize your Next.js and TinaCMS project with a single codebase
+  to efficiently manage multiple domains, locations, and content.
+uri: do-you-know-how-to-use-single-codebase-for-multiple-domains-with-tinacm-and-nextjs
+authors:
+  - title: ""
+created: 2024-08-28T17:37:00.000Z
+guid: 2bba887b-9dd7-4f07-978c-a793611e2f85
+---
+## Introduction
+
+This guide will walk you through the steps required to use a single codebase for multiple domains using TinaCMS and Next.js. This setup is particularly useful for managing content across different locations or websites, all from a centralized codebase. We will cover how to structure your project, configure middleware for domain-specific content, and manage environment variables for different locations.
+
```

Comment:
@PothieuG 

There doesn't seem to be a break i.e. <!--endintro--> to separate the intro from the main content

- [ ] Please fix